sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -61,15 +66,31 @@ export class ApplicationController extends BaseController {
         `Application id ${applicationId} was not found.`,
       );
     }
-
     return {
       data: application.data,
       id: application.id,
       applicationStatus: application.applicationStatus,
       applicationStatusUpdatedOn: application.applicationStatusUpdatedOn,
+      applicationNumber: application.applicationNumber,
+      applicationOfferingType: application.offering?.offeringIntensity,
```

Comment:
Is there a reason to change the name being returned from _offeringIntensity_ to _applicationOfferingType_?
Can we keep it the same?